src/probe_modules/module_dns.c

```diff
@@ -87,7 +87,9 @@ static uint16_t *dns_packet_lens; // Not including udp header
 static uint16_t *qname_lens;
 static char **qnames;
 static uint16_t *qtypes;
-static int num_questions = 0;
+static int num_questions = 0; // How many DNS questions to query. Note: There's a requirement that the num_questions = probes input by user
+static char probe_arg_delimitor = ';';
+static void find_num_dns_questions(struct state_conf *);
```

Comment:
It doesn't really feel like this "finds the number of questions" given that it doesn't return any integer value for the number of questions.

Reply:
That's very fair, I've updated it!